anyric/super-record
# #167615770 Integrate Roles to User Accounts

#### What does this PR do?
This PR add integration for user roles during account creation and update

#### Description of Task to be completed?
- Integrate user registration with roles

#### How should this be manually tested?
- Clone this branch
- In the root directory run `python manage.py runserver`
- Navigate to `http://127.0.0.1:8000/` to login
- Navigate to `http://127.0.0.1:8000/accounts/settings` 
- Select `Users` menu on the left, you should see users with their roles
- Click on the `Add User` button and there should be a roles dropdown on the form

#### Any background context you want to provide?
Currently the user registration and roles are not integrated as such roles can't be assign to users at registration or update. This should be rectified so that users are assigned role at registration and update.

#### What are the relevant pivotal tracker stories?
[#167615770](https://www.pivotaltracker.com/story/show/167615770)

#### Screenshots (if appropriate)
![image](https://user-images.githubusercontent.com/29800415/62390441-f8646280-b56a-11e9-93a1-00a3024d97c2.png)

![image](https://user-images.githubusercontent.com/29800415/62292454-1e0c4180-b46f-11e9-8e6e-cd638168af12.png)

![image](https://user-images.githubusercontent.com/29800415/62292477-25334f80-b46f-11e9-91f7-b24bbd839c39.png)


#### Questions:
N/A
